hack/checkout_account.py

```diff
@@ -27,24 +27,29 @@
 RESOURCE_TYPE = "aws-account"
 USER = "cluster-api-provider-aws"
 
-if __name__ == "__main__":
-    conn = httplib.HTTPConnection(BOSKOS_HOST)
-
+def post_request(input_state = "free"):
```

Comment:
I'd add a comment that we're working around an issue with the data in boskos and that we'll remove this code that tries both free and clean once all the data is good.